01-tar-instead-of-zip.patch

```diff
@@ -1,33 +1,37 @@
 diff --git a/tensorflow/lite/tools/make/download_dependencies.sh b/tensorflow/lite/tools/make/download_dependencies.sh
-index 0925adc6..ed640229 100755
+index 314f4fe6..9adc2d39 100755
 --- a/tensorflow/lite/tools/make/download_dependencies.sh
 +++ b/tensorflow/lite/tools/make/download_dependencies.sh
-@@ -31,17 +31,17 @@ fi
+@@ -35,7 +35,7 @@ fi
  
  EIGEN_URL="$(grep -o 'https.*gitlab.com/libeigen/eigen/-/archive/.*tar\.gz' "${BZL_FILE_PATH}" | grep -v mirror.tensorflow | head -n1)"
  EIGEN_SHA="$(eval echo $(grep '# SHARED_EIGEN_SHA' "${BZL_FILE_PATH}" | grep -o '\".*\"'))"
 -GEMMLOWP_URL="$(grep -o 'https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/gemmlowp/.*zip' "${BZL_FILE_PATH}" | head -n1)"
 +GEMMLOWP_URL="$(grep -o 'https://github.com/google/gemmlowp/.*tar.gz' "${BZL_FILE_PATH}" | head -n1)"
  GEMMLOWP_SHA="$(eval echo $(grep '# SHARED_GEMMLOWP_SHA' "${BZL_FILE_PATH}" | grep -o '\".*\"'))"
- GOOGLETEST_URL="https://github.com/google/googletest/archive/release-1.8.0.tar.gz"
+ RUY_URL="https://github.com/google/ruy/archive/91d62808498cea7ccb48aa59181e218b4ad05701.zip"
+ RUY_SHA="ac6d71df496a20043252f451d82a01636bb8bba9c3d6b5dc9fadadaffa392751"
+@@ -43,13 +43,13 @@ GOOGLETEST_URL="https://github.com/google/googletest/archive/release-1.8.0.tar.g
+ GOOGLETEST_SHA="58a6f4277ca2bc8565222b3bbd58a177609e9c488e8a72649359ba51450db7d8"
  ABSL_URL="$(grep -o 'https://github.com/abseil/abseil-cpp/.*tar.gz' "${BZL_FILE_PATH}" | head -n1)"
  ABSL_SHA="$(eval echo $(grep '# SHARED_ABSL_SHA' "${BZL_FILE_PATH}" | grep -o '\".*\"'))"
 -NEON_2_SSE_URL="https://github.com/intel/ARM_NEON_2_x86_SSE/archive/master.zip"
 +NEON_2_SSE_URL="https://github.com/intel/ARM_NEON_2_x86_SSE/archive/master.tar.gz"
  FARMHASH_URL="https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/farmhash/archive/816a4ae622e964763ca0862d9dbd19324a1eaf45.tar.gz"
  FARMHASH_SHA="$(eval echo $(grep '# SHARED_FARMHASH_SHA' "${BZL_FILE_PATH}" | grep -o '\".*\"'))"
- FLATBUFFERS_URL="https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/flatbuffers/archive/v1.11.0.tar.gz"
+ FLATBUFFERS_URL="https://github.com/google/flatbuffers/archive/v1.12.0.tar.gz"
+ FLATBUFFERS_SHA="62f2223fb9181d1d6338451375628975775f7522185266cd5296571ac152bc45"
  FFT2D_URL="https://storage.googleapis.com/mirror.tensorflow.org/www.kurims.kyoto-u.ac.jp/~ooura/fft2d.tgz"
 -FP16_URL="https://github.com/Maratyszcza/FP16/archive/febbb1c163726b5db24bed55cc9dc42529068997.zip"
 +FP16_URL="https://github.com/Maratyszcza/FP16/archive/febbb1c163726b5db24bed55cc9dc42529068997.tar.gz"
```

Comment:
just checking: last time we manually renamed the dependencies to `.zip` in order to fix build issues. Is that working better now?

Reply:
This was the previous file: https://github.com/totemic/conan-tensorflow-lite/blob/master/02-tar-instead-of-zip.patch
I think we were having issues with `unzip` not being on the current dockerized build pipeline, and changing this recipe was faster than creating new docker images.

This change is just an update to the patch. Not sure why the diff is messed up.

Reply:
Oh I see, it's the color highlighting of a `.patch` file that looks the same as the github diff.

Reply:
ah right, got confused by the "diff of a diff" display